email_recovery/lib/email_recovery/set/set_email_recovery_action_method.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:email_recovery/email_recovery/capability_deleted_messages_vault.dart';
+import 'package:email_recovery/email_recovery/email_recovery_action.dart';
+import 'package:jmap_dart_client/http/converter/id_converter.dart';
+import 'package:jmap_dart_client/http/converter/set/set_method_properties_converter.dart';
+import 'package:jmap_dart_client/jmap/core/capability/capability_identifier.dart';
+import 'package:jmap_dart_client/jmap/core/method/request/set_method.dart';
+import 'package:jmap_dart_client/jmap/core/request/request_invocation.dart';
+
+class SetEmailRecoveryActionMethod extends SetMethodNoNeedAccountId<EmailRecoveryAction> with OptionalCreate<EmailRecoveryAction> {
```

Comment:
`SetMethodNoNeedAccountId ` has been implemented `OptionalCreate`. So there's no need to `with` it anymore.